inspirehep/modules/references/view_utils.py

```diff
@@ -42,58 +40,20 @@ def references(self):
         out = []
         references = self.record.get('references')
         if references:
-            refs_to_get_from_es = [
-                ref['recid'] for ref in references if ref.get('recid')
-            ]
-            query = IQ(' OR '.join('recid:' + str(ref)
-                                   for ref in refs_to_get_from_es))
-            records_from_es = current_search_client.search(
-                index='records-hep',
-                doc_type='hep',
-                body={"query": query.to_dict()},
-                size=9999,
-                _source=[
-                    'control_number',
-                    'citation_count',
-                    'titles',
-                    'earliest_date',
-                    'authors',
-                    'collaboration',
-                    'corporate_author',
-                    'publication_info'
-                ]
-            )['hits']['hits']
-
-            refs_from_es = {
-                str(ref['_source']['control_number']): ref['_source'] for ref in records_from_es
-            }
-            for reference in references:
+            resolved_references = replace_refs(references, 'es')
```

Comment:
I remember that the reason for not doing this in the first place was because this would be one ES query per reference rather tan a single query for all.